user-service/src/main/java/uk/gov/gchq/palisade/service/user/UserClient.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.user;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import uk.gov.gchq.palisade.User;
+import uk.gov.gchq.palisade.service.user.request.GetUserRequest;
+
+@FeignClient(value = "user-service", url = "${web.client.user-service}")
+public interface UserClient {
+
+    @PostMapping(value = "/getUser", consumes = "application/json", produces = "application/json")
+    User getUser(final GetUserRequest request);
+
```

Comment:
It looks like this client is missing /addUser and /addUser/async - used in SimpleUserService

Reply:
Added  /addUser, but /addUser/async doesnt exist in SimpleUserService so not added